doc/installation/system/inifile.rst

```diff
@@ -83,6 +82,22 @@ slower but more robust and can be necessary in large redundant setups.
    row is otherwise not present". In this case setting ``PI_DB_SAFE_STORE``  to *True*
    might help.
 
+``PI_HASH_ALGO_LIST`` is a user-defined list of hash algorithms which are used
+to decrypt passwords and pins. The first entry in ``PI_HASH_ALGO_LIST`` is used
```

Comment:
`decrypt` is not the correct word here, we should use `verify`

Reply:
fixed in 809a696894b528cd75d443c12edb3af97dc922b6.